src/tests/server.test.ts

```diff
@@ -0,0 +1,23 @@
+import Server from '../modules/server'
+import request from 'supertest'
+
+const server = new Server()
+
+test('hello works', async () => {
+  server.start()
+  const response = await request((await server.getApp()).callback()).get('/hello')
```

Comment:
How about changing `supertest` to [http-client](https://www.npmjs.com/package/typescript-http-client) ?

Reply:
In this case, I think `supertest` works well since it can be easy mixed with different testing suites and also makes HTTP assertions easy. 

Refs:
https://blog.bitsrc.io/comparing-http-request-libraries-for-2019-7bedb1089c83
https://github.com/visionmedia/supertest
https://zellwk.com/blog/endpoint-testing/